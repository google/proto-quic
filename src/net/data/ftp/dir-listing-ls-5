drwxrwsr-x    4 501      501          4096 Feb 20  2007 pub
