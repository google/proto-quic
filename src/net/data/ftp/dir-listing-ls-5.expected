d
pub
-1
2007
2
20
0
0
